### ISSUE TITLE
(Not an issue) interactive 3ds viewer

### ISSUE THREAD

**Issue by Let0n** [harrispirie/stmpy#7]
Guys, I'm writing a python version of Nanonis 3ds viewer (such that it wouldn't only run in windows), not only for data but also if you would like to view data after analysis [e.g. Feenstra normalization dI/dV/(I/V)] or check fit data [e.g. a gap map]. If you've already had a similar code, please close this thread and let me know.
-- Jacky

**Comment by harrispirie** [comment]
Great idea! I haven't written anything similar; it sounds like a nice feature!

**Comment by Let0n** [comment]
Hi Harry and others,
I finished the first version of 3ds viewer in the subfolder 'mapviewer', and the main file is view3ds.py. Most of the features I have included is described in the helptext.txt or Help button when you run the GUI.

Feel free to try and submit issues here or email me. I show below doc string of the main function just for you to get a glimpse.

![Screen Shot 2021-02-01 at 9 39 33 PM](https://user-images.githubusercontent.com/22223289/106515460-02f01700-64d6-11eb-89b2-e32b3cee1497.png)
